core/trino-main/src/main/java/io/trino/operator/ScanFilterAndProjectOperator.java

```diff
@@ -164,7 +164,7 @@ public long getDynamicFilterSplitsProcessed()
     }
 
     @Override
-    public Metrics getConnectorMetrics()
```

Comment:
This was intentionally named "connector metrics" to distinguish it from other metrics that may be provided by the operator itself (the filter/project part)

Reply:
Ok. I will fix this in follow-up PR

Reply:
@martint here is follow-up https://github.com/trinodb/trino/pull/9156